CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing
+
+When contributing to this repository, please first check whether the issue has already been logged, and if not, please file the ticket by creating a New Issue.
+
+## Logging an Issue
+
+1. Make sure this issue has not already been raised. If a relevant issue has been closed already please reopen it or make a reference to it in a new issue.
+2. Add appropriate labels to the issue (e.g. F1-documentation)
+3. Add the issue to the appropriate Project and/or Milestones. If unsure, leave it blank.
+4. For minor issues, it is acceptable to describe the problem and offer a potential route to fix it. For more involved feature requests or bugs, please include the following details at the minimum:
+
+##### For bugs:
+
+- current system OS
+- repository branch
+- steps to reproduce
+- stack trace
+
+##### For feature requests:
+
+- rationale
+- paint the feature step by step (in words)
+- mockup (optional)
+
+## Pull Request Process
+
+1. Pick an issue off the Issue tracker for the repository, and assign yourself before working on it so we don't have duplicated effort.
+2. If unsure about the specifics of implementing a particular issue, please make a Draft PR sooner rather than later, and start a discussion from there.
+3. When designing a new UI component, please make sure your changes are also reflected in the appropriate Storybook story.
+4. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+5. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
+   do not have permission to do that, you may request the second reviewer to merge it for you.
+
+## Commit Messages
+
+Please follow the standards outlined in Conventional Commits: https://www.conventionalcommits.org/en/v1.0.0-beta.3/#specification
+
+## Code of Conduct
+
+### Our Pledge
```

Comment:
This was much needed. I can't belive how we are running our open source project without code of conduct. Thankyou very much for that.